src/dvclive/live.py

```diff
@@ -446,6 +446,33 @@ def log_artifact(
                     )
 
     def cache(self, path):
+        if self._inside_dvc_exp:
+            msg = f"Skipping dvc add {path} because `dvc exp run` is running."
+            path_stage = None
+            for stage in self._dvc_repo.index.stages:
+                for out in stage.outs:
+                    if out.fspath == str(Path(path).absolute()):
+                        path_stage = stage
+                        break
+            if not path_stage:
```

Comment:
Maybe it was just my bias from knowing the previous implementation. Still, I ended up with the model tracked in Git because we are skipping the `dvc add` entirely, which was previously adding it to gitignore.
I think we might warning but not skip the `dvc add`, as if we reach this point it means that the user is not tracking it in the `dvc.yaml` and the is no `.dvc` file

Reply:
Yeah, this was probably a bit overly aggressive. I'll submit a PR to keep trying `dvc add` unless it's already tracked in the pipeline.